kubric/core/objects.py

```diff
@@ -84,6 +86,8 @@ class PhysicalObject(Object3D):
   friction = tl.Float(0.0)
   restitution = tl.Float(0.5)
 
+  # TODO: a tuple of two numpy arrays is annoying to work with
+  #       either convert to single 2D array or to as tuples
   bounds = tl.Tuple(ktl.Vector3D(), ktl.Vector3D(),
```

Comment:
what's "bounds" can you add a docstring to PhysicalObject?

Reply:
It's 2x 3D vectors... so I assume this is an AABB? (axis aligned bounding box)?
If that's the case, shouldn't we be more explicit and make a type for it?

Reply:
Yes that would be a reasonable option. I was just too lazy to tidy this up ATM. Hence the TODO